shared/AppInsightsCore/src/JavaScriptSDK/ChannelController.ts

```diff
@@ -1,274 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// // Licensed under the MIT License.
-
-import { ITimerHandler, arrForEach, isArray, objFreeze, scheduleTimeout, throwError } from "@nevware21/ts-utils";
```

Comment:
Ok, I think I've managed to re-instate the git mv to keep the history of this file -- will publish an update shortly

Reply:
:-( Ok, local shows this as a git move/rename but here as a delete and add...